src/store/wallet.js

```diff
@@ -50,18 +56,40 @@ export default {
   mutations: {
     setTransactions(state, { transactions }) {
       state.transactions = transactions.map(transaction => {
-        let date = new Date(transaction.timestamp * 1000)
+        let transactionType = null
```

Comment:
IMO we should put the logic that normalizes the data from the wallet in api.js

Reply:
@Tommytrg what do you mean?

Reply:
Move the normalization logic to a different file in order to reduce `wallet.js`